tools/spec-gen-sdk/src/cli/cli.ts

```diff
@@ -173,7 +176,7 @@ yargs(hideBin(process.argv))
           description: "The branch of the head repository of the specification pull request",
         },
         'api-version': {
-          alias: "v",
+          alias: "apiv",
```

Comment:
You could just make it av

Reply:
I intended not using 'av' for potential Policheck alert.